sal_interfaces/HVAC/HVAC_Commands.xml

```diff
@@ -1,30 +1,317 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl"?>
-<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
-<SALCommand>
-    <Subsystem>HVAC</Subsystem>
-    <Version>3.8</Version>
-    <Author></Author>
-    <EFDB_Topic>HVAC_command_setHvacConfiguration</EFDB_Topic>
-    <Alias>setHvacConfiguration</Alias>
-    <Device></Device>
-    <Property></Property>
-    <Action></Action>
-    <Value></Value>
-    <Explanation>http://sal.lsst.org</Explanation>
-    <item>
-        <EFDB_Name>maxValveFlow</EFDB_Name>
-        <Description>Maximum flow.</Description>
-        <IDL_Type>double</IDL_Type>
-        <Units>m3 / s</Units>
-        <Count>1</Count>
-    </item>
-    <item>
-        <EFDB_Name>minValveFlow</EFDB_Name>
-        <Description>Minimum flow.</Description>
-        <IDL_Type>double</IDL_Type>
-        <Units>m3 / s</Units>
-        <Count>1</Count>
-    </item>
-</SALCommand></SALCommandSet>
\ No newline at end of file
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type='text/xsl' href='http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl'?>
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
+  <SALCommand>
+    <Subsystem>HVAC</Subsystem>
+    <EFDB_Topic>HVAC_command_lsstBarraoblPiso01BarraoblTccGuionP1GuionSalaGuionMaquinas</EFDB_Topic>
+    <Alias>lsstBarraoblPiso01BarraoblTccGuionP1GuionSalaGuionMaquinas</Alias>
+    <item>
+      <EFDB_Name>lsstBarraoblPiso01BarraoblVea_01BarraoblComandoEncendidoLsst</EFDB_Name>
```

Comment:
Is there an easy way to get rid of these remaining underscores?

Reply:
Possibly yes. I'll open an issue for it because I will need to think about how to do that without introducing too much "noise".